workbench/about.html

```diff
@@ -1,129 +1,147 @@
-    </p>
+    <div id="links">
+      <p>Documentation: 
+        <a
+          href="http://releases.naturalcapitalproject.org/invest-userguide/latest/"
+        >
+          http://releases.naturalcapitalproject.org/invest-userguide/latest/
+        </a>
+      </p>
+      <p>Homepage: 
+        <a
+          href="https://naturalcapitalproject.stanford.edu/"
+        >
+          https://naturalcapitalproject.stanford.edu/
+        </a>
+      </p>
+      <p>Project page: 
+        <a
+          href="https://github.com/natcap/invest"
+        >
+          https://github.com/natcap/invest
+        </a>
+        </p>
+      <p>License: 
+        <a
+          href="https://github.com/natcap/invest/blob/master/LICENSE.txt"
+        >
+          BSD 3-clause
+        </a>
+      </p>
+    </div>
+    <div id="licenses">
+      <h4>Open-Source Licenses:</h4>
+      <table>
+          <tr>
+            <td>PyInstaller</td>
+            <td>GPL</td>
+            <td>
+              <a
+                href='http://pyinstaller.org'
+              >
+                http://pyinstaller.org
+              </a>
+            </td>
+          </tr>
+          <tr>
+            <td>GDAL</td>
+            <td>MIT and others</td>
+            <td>
+              <a
+                href='http://gdal.org'
+              >
+                http://gdal.org
+              </a>
+            </td>
+          </tr>
+          <tr>
+            <td>numpy</td>
+            <td>BSD</td>
+            <td>
+              <a
+                href='http://numpy.org'
+              >
+                http://numpy.org
+              </a>
+            </td>
+          </tr>
+          <tr>
+            <td>pygeoprocessing</td>
+            <td>BSD</td>
+            <td>
+              <a
+                href='https://github.com/natcap/pygeoprocessing'
+              >
+                https://github.com/natcap/pygeoprocessing
+              </a>
+            </td>
+          </tr>
+          <tr>
+            <td>rtree</td>
+            <td>LGPL</td>
+            <td>
+              <a
+                href='http://toblerity.org/rtree/'
+              >
+                http://toblerity.org/rtree/
+              </a>
+            </td>
+          </tr>
+          <tr>
+            <td>scipy</td>
+            <td>BSD</td>
+            <td>
+              <a
+                href='http://www.scipy.org/'
+              >
+                http://www.scipy.org/
+              </a>
+            </td>
+          </tr>
+          <tr>
+            <td>shapely</td>
+            <td>BSD</td>
+            <td>
+              <a
+                href='http://github.com/Toblerity/Shapely'
+              >
+                http://github.com/Toblerity/Shapely
+              </a>
+            </td>
+          </tr>
+      </table>
+      <p>
+        <em>The source code for GPL'd components are included as an extra
+        component on your installation medium.</em>
```

Comment:
Yeah, we should probably do this, though maybe linking to the software projects themselves is sufficient?

Technically speaking, we're responsible for making available the source code of any GPLed software we distribute, so including them in the installer would be a nice way to fulfill our obligations under the GPL for everyone obtaining the binaries.  But it also adds a lot of overhead to actually include all that software.  Just in a quick spot-check of what other projects do, I see slack links to a [page with their dependencies](https://slack.com/libs/desktop), and Atom includes a textfile with everything concatenated together, along with the node package that it references.  So maybe what we have in the dialog is sufficient?

Do we distribute GPL'd source code in the workbench's distribution medium?

Although we might be obligated to do this (and I know the Qt interface says we do this), we might want to not include this extra sentence until we do offer it.

Reply:
I also just noticed we're not including our `LICENSE.txt` in the distribution. It's linked from from the About page, but should probably be included in the distribution.

Reply:
Thanks for including our LICENSE!  I just created an issue for generating a list of python packages here: https://github.com/natcap/invest/issues/979, as we were talking about on the coffee call this morning.